### ISSUE TITLE
make yin yang work like statue park

### ISSUE THREAD

**Issue by robx** [robx/pzprjs#295]
I've always found the yin yang player to be a bit off:

- there's something a bit unexpected about the input mode
- the visuals are quite confusing

Statue park on the other hand feels just right, and it's basically the same thing. So I'd like to switch yin yang over to behave pretty much the same. (Circle size might better stay the way it is, or a bit larger than statue park at any rate? Not sure.)

**Comment by x-sheep** [comment]
I fully agree on the input being tedious, but changing the answer visuals would go against how it's solved everywhere else.
